packages/core-interop/src/core-interop/Localization.ts

```diff
@@ -0,0 +1,14 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+import { Localization } from "@itwin/core-common";
+
+/**
+ * Create a `Translator` that uses [Localization]($core-common) API to create a localized string.
+ * @beta
+ */
+export function createTranslator(localization: Localization): (input: string) => string {
+  return (input) => localization.getLocalizedString(input);
```

Comment:
I believe here we could take care of registering the `hierarchy-builder` localization namespace (that would mean `createTranslator` has to become `async`).

Reply:
Added